### PR TITLE
Merge backend scaffold into SFTT

### PR DESCRIPTION
**Pull request by liammoyn**

### CODE REVIEW

**Comment by liammoyn** [comment]
Is it worth squashing all of these commits together? I don't think that would mess anything up

File: api/src/main/java/com/codeforcommunity/rest/subrouter/CommonRouter.java [hunk 1]
```diff
@@ -24,8 +25,8 @@ public CommonRouter(JWTAuthorizer jwtAuthorizer) {
   public Router initializeRouter(Vertx vertx) {
     Router router = Router.router(vertx);
 
+    router.route().handler(LoggerHandler.create()); // Adds request logging
```
Comment (florisdobber) [inline api/src/main/java/com/codeforcommunity/rest/subrouter/CommonRouter.java:28]: Where does it log requests? This is not the Slack notification thing right?

Comment (liammoyn) [reply api/src/main/java/com/codeforcommunity/rest/subrouter/CommonRouter.java:28]: No this just prints to console whenever an incoming request is made

**Review by connernilsen — APPROVED** [review]
Looks good

**Comment by connernilsen** [comment]
Double check that the tests actually ran in the builds though

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # C4C Backend Scaffold
 
 [![Build Status](https://travis-ci.org/Code-4-Community/backend-scaffold.svg?branch=master)](https://travis-ci.org/Code-4-Community/backend-scaffold)
-[![Coverage Status](https://coveralls.io/repos/github/Code-4-Community/backend-scaffold/badge.svg?branch=add-coveralls)](https://coveralls.io/github/Code-4-Community/backend-scaffold?branch=add-coveralls)
+[![Coverage Status](https://coveralls.io/repos/github/Code-4-Community/backend-scaffold/badge.svg?branch=master)](https://coveralls.io/github/Code-4-Community/backend-scaffold?branch=master)
```
Comment (jackblanc) [inline README.md:4]:
```suggestion
[![Coverage Status](https://coveralls.io/repos/github/Code-4-Community/speak-for-the-trees/badge.svg?branch=master)](https://coveralls.io/github/Code-4-Community/speak-for-the-trees?branch=master)
```

**Review by jackblanc — APPROVED** [review]
Looks good 👍 one small change in readme for the repo name

**Pull request merged**